en/deploy-heterogeneous-tidb-cluster.md

```diff
@@ -0,0 +1,173 @@
+
+### Create a heterogeneous cluster
+
+Save the following configuration as the `cluster.yaml` file. Replace `<heterogeneous_cluster_name>` with the desired name of your heterogeneous cluster, and replace `<origin_cluster_name>` with the name of the existing cluster.
+
+{{< copyable "shell-regular" >}}
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: ${heterogeneous_cluster_name}
+spec:
+  configUpdateStrategy: RollingUpdate
+  version: v4.0.6
+  timezone: UTC
+  pvReclaimPolicy: Delete
+  discovery: {}
+  cluster:
+    name: ${origin_cluster_name}
+  tikv:
+    baseImage: pingcap/tikv
+    replicas: 1
+    # if storageClassName is not set, the default Storage Class of the Kubernetes cluster will be used
+    # storageClassName: local-storage
+    requests:
+      storage: "1Gi"
+    config: {}
+  tidb:
+    baseImage: pingcap/tidb
+    replicas: 1
+    service:
+      type: ClusterIP
+    config: {}
+  tiflash:
+    baseImage: pingcap/tiflash
+    maxFailoverCount: 1
+    replicas: 1
+    storageClaims:
+      - resources:
+          requests:
+            storage: 1Gi
+        storageClassName: standard
+```
+
+Execute the following command to create the heterogeneous cluster:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl create -f cluster.yaml -n ${namespace}
+```
+
+The configuration of a heterogeneous cluster is mostly the same as a normal TiDB cluster, except that it uses the `spec.cluster.name` field to join the target cluster.
+
+### Deploy the cluster monitoring component
+
+Save the following configuration as the `tidbmonitor.yaml` file. Replace `<heterogeneous_cluster_name>` with the desired name of your heterogeneous cluster, and replace `<origin_cluster_name>` with the name of the existing cluster.
+
+{{< copyable "shell-regular" >}}
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbMonitor
+metadata:
+  name: heterogeneous
+spec:
+  clusters:
+    - name: ${origin_cluster_name}
+    - name: ${heterogeneous_cluster_name}
+  prometheus:
+    baseImage: prom/prometheus
+    version: v2.11.1
+  grafana:
+    baseImage: grafana/grafana
+    version: 6.1.6
+  initializer:
+    baseImage: pingcap/tidb-monitor-initializer
+    version: v4.0.6
+  reloader:
+    baseImage: pingcap/tidb-monitor-reloader
+    version: v1.0.1
+  imagePullPolicy: IfNotPresent
+```
+
+Execute the following command to create the heterogeneous cluster:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl create -f tidbmonitor.yaml -n ${namespace}
+```
+
+## Deploy a TLS-enabled heterogeneous cluster
+
+To enable TLS for a heterogeneous cluster, you need to explicitly declare TLS configuration, create a new `Secret` certificate, and issue the certificate using the same certification authority (CA) as the target cluster.
+
+If you want to issue the certificate using `cert-manager`, choose the same `Issuer` as the target cluster to create your `Certificate`.
```

Comment:
```suggestion
If you want to issue the certificate using `cert-manager`, choose the same `Issuer` as that of the target cluster to create your `Certificate`.
```

Reply:
is it better to move L170 & 171 to this section and state that users can refer to those two procedures to create certs for the heterogeneous cluster?

Reply:
Sure, updated.